stdpopsim/species.py

```diff
@@ -236,3 +243,17 @@ def get_genetic_map(self, id):
             if gm.id == id:
                 return gm
         raise ValueError(f"Genetic map '{self.id}/{id}' not in catalog")
+
+    def add_annotations(self, annotations):
```

Comment:
i don't understand what this is doing

Reply:
this is adding to the catalog of possible annotations in a Species